builders/trailsBuilder.js

```diff
@@ -66,6 +66,38 @@ const queries = [
       WHERE (type = 'hike' OR type = 'horse' OR type = 'bike') AND name != ''
     `
   },
+  {
+    name: "trail-endpoints",
+    minZoom: 9,
+    maxZoom: 10,
+    query: `
+      WITH filtered_trails as (
+        SELECT geog::geometry as geom from trails
+        WHERE ST_Length(geog) > 3000 AND type = 'hike' OR type = 'horse' OR type = 'bike' OR
```

Comment:
An _IN_ clause could work well here.  

Ex.  _AND type in ('hike', 'horse', 'motorcycle', 'atv')_
Another Ex. _AND type in (SELECT filters FROM trailtypefilters GROUP BY filters)_